src/service/utils.js

```diff
@@ -2,29 +2,130 @@ const fs = require('node:fs/promises')
 const { createReadStream, createWriteStream } = require('node:fs')
```

Comment:
it is probably my java deformation but rather than naming this utils i would name `http-request-static-functions` or if you really like utils `http-request-utils-functions`